redirects/2.15/api/applications/convnext/index.html

```diff
@@ -0,0 +1 @@
+<meta http-equiv="refresh" content="0; URL='https://keras.io/2/api/applications/convnext/'" />
```

Comment:
Is the `/2/` supposed to be there? This URL doesn't work for me.

Reply:
It will work after this PR is merged.